### ISSUE TITLE
Distributed Docker compose with MariaDB nodes can't communicate each others

### ISSUE THREAD

**Issue by mliradelc** [opencast/opencast-docker#110]
I just tested this version and I have some issues:

The worker, presenter and the and the admin can't see each other because the URLs are configured without a proper alias or the internal docker network.

**Comment by mtneug** [comment]
I could not reproduce the issues. Docker compose will automatically create a Docker network with a DNS server that resolves the `opencast-{admin|worker|presentation}` domains to the correct IPs.

Using the `next` branch for the upcoming Opencast 7, I run the following commands:

```sh
$ cd path/to/repo
$ docker-compose -p opencast-multiserver -f docker-compose/docker-compose.multiserver.mariadb.yml up
```

Could you describe your setup in more detail?

**Comment by mliradelc** [comment]
I'm using the official docker for mac, the version is `Docker version 18.09.2, build 6247962`

after that, I cloned the repo and use the "next" branch with the same command as you suggested.

The first problem I had is the exposed port for MariaDB, that caused a conflict with my installed MariaDB, so I commented the line  `- 3306:3306` of the MariaDB configuration.

Later I have the network problem I described here, but I just solved expressing the aliases for the default network like this for each container:

```yaml
    networks:
      default:
        aliases:
          - activemq
```

@mtneug you have right, the docker compose creates a network, but for any reason didn't work for me until I explicitly configured the aliases.
